### ISSUE TITLE
Does not learn the distance well on my own dataset

### ISSUE THREAD

**Issue by ckLibra** [tinghuiz/SfMLearner#126]
Like the following picture.The network seems to learn the structure well,but the distance is not learned well.The depth in the distance seems to be similar to that in the near place.What's the reason that might influence it?
I use the pretrained model to finetune.Hyperparameter is all default except exp_mask_weight=0.2 and epochs=20.
And I add an corner mask for photometric_reconstruction_loss since the four corner in my dataset is all black and meaningless.
![1](https://user-images.githubusercontent.com/30599093/78625438-a70d2f80-78be-11ea-8fbc-aee1c1632c6e.png)
![2](https://user-images.githubusercontent.com/30599093/78625441-a83e5c80-78be-11ea-848f-c6c0d52c3775.png)
